### DISCUSSION TITLE
音频文件如何实现倍速播放啊

### DISCUSSION THREAD

**Discussion by emotinapua** [spencerwooo/onedrive-vercel-index#629] [Q&A]
![image](https://user-images.githubusercontent.com/104719318/166175346-d008c4be-9c15-49a1-aca6-7a60306a4464.png)

就像这样，没有倍速播放，如何实现呢

**Comment by myl7** [comment]
音频播放使用的是 [react-audio-player](https://www.npmjs.com/package/react-audio-player)，似乎没有明确的倍速播放支持。所以应该没法很简单地添加此功能。
